luda-editor/client/src/components/name_quick_slot.rs

```diff
@@ -0,0 +1,54 @@
+use crate::app::notification::{push_notification, Notification};
+use namui::prelude::*;
+use std::collections::HashMap;
+
+#[derive(Clone, Debug)]
+pub struct NameQuickSlot {
+    sequence_id: Uuid,
+    names: HashMap<usize, String>,
+}
+impl NameQuickSlot {
+    pub async fn load_from_cache(sequence_id: Uuid) -> anyhow::Result<Self> {
+        cache::get_serde(&Self::cache_key(sequence_id))
```

Comment:
quick slot should be cached by project, not sequence. Because it's very common to use same characters among the sequences in project.

Reply:
Fixed 4ac5dc4